x/qgb/keeper/query_valset_confirm.go

```diff
@@ -25,3 +26,27 @@ func (k Keeper) ValsetConfirmsByNonce(
 	confirms := k.GetValsetConfirms(sdk.UnwrapSDKContext(c), req.Nonce)
 	return &types.QueryValsetConfirmsByNonceResponse{Confirms: confirms}, nil
 }
+
+const maxValsetRequestsReturned = 5
```

Comment:
Shouldn't we create a specific package containing the consts ?

Reply:
that's a good idea, tbh this was just copied over from the gravity module.

Reply:
created issue: https://github.com/celestiaorg/celestia-app/issues/262

Reply:
Thinking about this more, while it is a constant, it is not exported, so I'm not sure how much the net gain is by moving it. It seems like it is meant to be a local constant imho. That being said, it is still in a weird spot, as constants typically are found at the top of a go file. Moved it there for now, but still keeping #262 open so we can think about this more as we continually work on this module.